.eslintrc.json

```diff
@@ -2,7 +2,8 @@
     "extends": "airbnb",
     "parser": "babel-eslint",
     "env": {
-        "es6": true
+        "es6": true,
+        "jest": true
```

Comment:
👍 let's do that, it will make more sense since `env:jest` is not true most of the time